cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/validation/ValidatingInputFormatProvider.java

```diff
@@ -18,27 +18,27 @@
 
 import io.cdap.cdap.api.data.batch.InputFormatProvider;
 import io.cdap.cdap.api.data.schema.Schema;
-import io.cdap.cdap.etl.api.FailureCollector;
 
 import javax.annotation.Nullable;
 
 /**
  * Validating input format provider.
  */
 public interface ValidatingInputFormatProvider extends InputFormatProvider {
+  String PLUGIN_TYPE = "validatingInputFormatProvider";
```

Comment:
this seems overly verbose, maybe just validatingInputFormat? similar comment for the interface name.